v22.1/follower-reads.md

```diff
@@ -1,197 +1,191 @@
 
-## Exact staleness reads
+- _Exact staleness read_: A historical read as of a static, user-provided timestamp. See [Exact staleness reads](#exact-staleness-reads).
+- _Bounded staleness read_: A historical read that uses a dynamic, system-determined timestamp to minimize staleness while being more tolerant to replication lag than an exact staleness read. See [Bounded staleness reads](#bounded-staleness-reads).
 
-An _exact staleness read_ is a historical read as of a static, user-provided timestamp. Most users will get an exact staleness read by using the timestamp value returned by the `follower_read_timestamp()` convenience [function](functions-and-operators.html). For an example showing how to use this function to get an exact staleness read, see [Run queries that use exact staleness reads](#run-queries-that-use-exact-staleness-follower-reads).
+### Exact staleness reads
 
-### When to use exact staleness reads
+An _exact staleness read_ is a historical read as of a static, user-provided timestamp.
 
-Use [exact staleness](#exact-staleness-reads) follower reads when:
+For requirements and limitations, see [Exact staleness reads and long-running writes](#exact-staleness-reads-and-long-running-writes) and [Exact staleness read timestamps must be far enough in the past](#exact-staleness-read-timestamps-must-be-far-enough-in-the-past).
 
-- You need multi-statement reads inside transactions.
-- You can tolerate reading older data (at least 4.8 seconds in the past), to reduce the chance that the historical query timestamp is not quite old enough to prevent blocking on a conflicting write and thus being able to be served by a local replica.
-- You do not need the increase in availability provided by [bounded staleness reads](#bounded-staleness-reads) in the face of network partitions or other failures.
-- You need a read that is slightly cheaper to perform than a [bounded staleness read](#bounded-staleness-reads), because exact staleness reads don't need to dynamically compute the query timestamp.
+#### When to use exact staleness reads
 
-You should **not** use follower reads when your application cannot tolerate reading stale data, since the results of follower reads may not reflect the latest writes against the tables you are querying. However, for many applications, especially in [multi-region deployments](multiregion-overview.html), there are opportunities to do useful work with follower reads.
+Use exact staleness follower reads when you:
 
-## Bounded staleness reads
+- Need multi-statement reads inside transactions.
+- Can tolerate reading older data (at least 4.8 seconds in the past), to reduce the chance that the historical query timestamp is not quite old enough to prevent blocking on a conflicting write and thus being able to be served by a local replica.
+- Do not need the increase in availability provided by [bounded staleness reads](#bounded-staleness-reads) in the face of network partitions or other failures.
```

Comment:
"network partitions" could link to https://www.cockroachlabs.com/docs/v21.2/cluster-setup-troubleshooting.html#network-partition